src/msw/webview_ie.cpp

```diff
@@ -34,6 +34,10 @@
 #include <initguid.h>
 #include <wininet.h>
 
+#ifndef DOCHOSTUIFLAG_ENABLE_REDIRECT_NOTIFICATION
+#define DOCHOSTUIFLAG_ENABLE_REDIRECT_NOTIFICATION  0x04000000
+#endif
```

Comment:
This should be defined in `include/wx/msw/webview_missing.h` along with the other flags.